sickbeard/providers/scenetime.py

```diff
@@ -87,7 +85,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     logger.log('Search string: {search}'.format
                                (search=search_string), logger.DEBUG)
 
-                search_url = self.urls['search'] % (quote(search_string), self.categories)
+                search_url = self.urls['search'].format(search_string, self.categories)
```

Comment:
I know params would be a bigger change, but I think leaving the quoting in would be a good idea until the param change as it would only be an issue on occasion and thus may not have been encountered during testing.